### PR TITLE
Words counter

### PR DESCRIPTION
**Pull request by Ragsomar**

### CODE REVIEW

File: david_goodedit/src/components/Tutorial.jsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ import QuoteTuto from './QuoteTuto'
 
 function Tutorial() {
   return (
-    <div>
-      <h1 className="titleSection">Tutoriel</h1>
+    <div className=':root'>
```
Comment (rocambille) [inline david_goodedit/src/components/Tutorial.jsx]: :root is not a class name: it is applied to root element of the document... and inherited by every elements, so no need to put it here :)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
there are conflicts to resolve :/

**Comment by ghost** [comment]
Still lots of conflicts here

**Comment by Ragsomar** [comment]
Yeah I close this one to open another with less conflicts (maybe)

**Pull request closed without merging**